cli/src/index.ts

```diff
@@ -42,8 +42,10 @@ type Args = {
   logLevel?: LogLevel;
   toolName?: string;
   toolArg?: Record<string, JsonValue>;
+  toolMeta?: Record<string, string>;
```

Comment:
As in the web client, `toolMeta` as a separate metadata package from `metaData` is not being exercised anywhere, so it's hard to see why it's in the PR.